stories/internal/DualAxesChartConfig.tsx

```diff
@@ -27,45 +28,49 @@ const MinMaxInfo = ({
     </div>
 );
 
-const renderSupportedCharts = (dataset: any, config?: IChartConfig, minmaxInfo = {}) => (
-    <div>
-        {BASE_DUAL_AXIS_CHARTS.map(type => {
-            const _config: IChartConfig = {
-                type,
-                legend: {
-                    enabled: true,
-                    position: "top",
-                },
-                secondary_yaxis: {
-                    measures: ["wonMetric"],
-                },
-                ...config,
-            };
-            const style = {
-                height: 600,
-                width: 600,
-            };
-
-            if (type === VisualizationTypes.BAR) {
-                _config.xaxis = _config.yaxis;
-                _config.secondary_xaxis = _config.secondary_yaxis;
-                _config.yaxis = _config.secondary_yaxis = undefined;
-                _config.dataLabels = {
-                    visible: false, // disable data label on bar chart to make test stable
+const renderSupportedCharts = (dataset: any, config?: IChartConfig, minmaxInfo = {}) =>
+    screenshotWrap(
+        <ScreenshotReadyWrapper resolver={createHighChartResolver(3)}>
+            {BASE_DUAL_AXIS_CHARTS.map((type: ChartType, index: number) => {
+                const _config: IChartConfig = {
```

Comment:
Typescript by default is allowing `_` to mark unused function parameter.
So please remove `_` for this variable and you might change `config` parameter to other name like `chartConfig`

Reply:
fixed